packages/flame/lib/src/devtools/connectors/game_loop_connector.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:convert';
+import 'dart:developer';
+
+import 'package:flame/game.dart';
+import 'package:flame/src/devtools/dev_tools_connector.dart';
+import 'package:flutter/foundation.dart';
+
+/// The [GameLoopConnector] is responsible for reporting and setting the
+/// pause/running state of the game and stepping the game forwards or backwards
+/// from the devtools extension.
+class GameLoopConnector extends DevToolsConnector {
+  var _pauseNotifier = ValueNotifier<bool>(true);
```

Comment:
This can't be `late final` since it will be set to a new `ValueNotifier` in `initGame`, it could probably be late now though!

Reply:
Ah no, it needs a default since the calls from the devtools might come in before the game is initialized, and it'd just be annoying to handle null too.